.eslintrc.json

```diff
@@ -145,6 +145,14 @@
           "off"
         ]
       }
+    },
+    {
+      "files": [
+        "tests/e2e/**/*.js"
+      ],
+      "rules": {
+          "react-hooks/rules-of-hooks": "off"
```

Comment:
We already have a rule group above for `tests/e2e/**/*.js` so we can merge this one into  that existing definition.